docs/config.md

```diff
@@ -0,0 +1,96 @@
+# Config file options
+
+SOCI has a multitude of variables that can customize SOCI behavior to best suit users' needs. This doc will break down the possible values for each variable, where the variable can be found, and what it can do.
+
+This doc assumes the usage of a TOML file. Per TOML specs, Anything in [brackets] is the required header for the set of variables.
+
+An example configuration file can be found in config/config.toml.
```

Comment:
I don't think that's possible, since actually navigating there would be up to however you're opening the project view. But I could link to the GitHub page?

Reply:
Github handles relative paths correctly (e.g.)
https://github.com/awslabs/soci-snapshotter/blob/eaf3231606368c675561281e572c275f295f1ba3/docs/build.md?plain=1#L140

Reply:
Done.